notebooks/ai-ml/HuggingFaceEmbeddingGenerationInSpark.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    prediction_filtered = prediction.filter(size(prediction["embeddings"]) == 384)
</pre><p>Let's add a comment explaining the reasoning, this 384 would be specific to the model used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>